index.html

```diff
@@ -345,14 +341,46 @@ <h3>Skills</h3>
               <li class="skill-tag skill-tag--white">Gitlab</li>
               <li class="skill-tag skill-tag--white">Terminal</li>
             </ul>
-
           </div>
         </div>
-
       </div>
-
     </section>
   </main>
+  <footer>
+    <div class="footer-container row">
+      <div class="form-container">
+        <h2 class="heading-secondary heading-secondary--blue">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/myyvwvbw" method="POST">
+          <input type="text" name="name" id="name" placeholder="Full name" required />
```

Comment:
- [ ] Kindly add `maxlength="30"` attribute to the name field so it can only accept up to 30 characters. 😄